rollup.config.js

```diff
@@ -2,6 +2,7 @@ import esbuild from "rollup-plugin-esbuild";
 import nodePolyfills from "rollup-plugin-polyfill-node";
 import { nodeResolve } from "@rollup/plugin-node-resolve";
 import commonjs from "@rollup/plugin-commonjs";
+import json from "@rollup/plugin-json";
```

Comment:
What's the context for this change?

Reply:
It wouldn't build without it, very confusing. Going to look more deeply. This was my initial solution

Reply:
It refuses to build without it, can't pin down the main issue. Is it a blocker?